### PR TITLE
added text component

### PR DESCRIPTION
**Pull request by Valentine-Mario**

### CODE REVIEW

File: src/forms/mod.rs [hunk 1]
```diff
@@ -1 +1,2 @@
 pub mod controls;
+pub mod text;
```
Comment (cecton) [inline src/forms/mod.rs:2]:
```suggestion
pub mod text;
```

Comment (cecton) [inline src/forms/mod.rs:2]: missing new line at the end of the file

**Comment by cecton** [comment]
Can you also put a cross in the README for <Text>?

File: src/forms/mod.rs (hunk 1 above)
Comment (Valentine-Mario) [reply src/forms/mod.rs]: added in new commit

**Comment by cecton** [comment]
Do you want to join the maintainers? (Much development needed, you will be able to review and merge PRs.) We have a discord channel: https://discord.gg/bqhv9T

(CI is still broken but I'll fix it)

**Comment by cecton** [comment]
Hey sorry I had to revert. The component is more broken than I thought and it is not what it seems (it's not the <Text> component of blueprint, it looks more like the Text area or an inputfield). If you make another PR I will review better.

**Pull request merged**